frontend/public/learn/Interactive-UI-Elements.elm

```diff
@@ -85,36 +92,38 @@ This is a very high level view of how things work, but it illustrates the
 typical structure of interactive UI elements in Elm. Let's look at the API
 in more detail.
 
-## Inputs
+## Channels
 
-The first thing we do when making an interactive UI element is to create an input:
+The first thing we do when making an interactive UI element is to create a channel:
 
 ```haskell
-type Input a = { signal : Signal a, handle : Handle a }
+type Channel a = Channel
 ```
 
-An `Input` has two distinct parts. One is a `signal` of events coming from UI
+A `Channel` has two distinct uses. One is to provide a `Signal` of events coming from UI
 elements. In our synced checkbox example, these are boolean values indicating
-whether the box should be checked or not. The more subtle part of an `Input` is
-the `handle`. All interactive UI elements will latch on to a handle and report
-their events to the corresponding `Input`. The handle lets us get events from
+whether the box should be checked or not. This `Signal` can be subscribed to. The more subtle part of a `Channel` is
+to provide a `Signal.Message` handler. All interactive UI elements will latch on to a message handler and so report
+their events to the corresponding `Channel`. The message handler lets us get events from
 the UI back into our program without any messy callbacks or event-listeners.
```

Comment:
Maybe switch the order of the uses. Handler first, then subscribe. That way, we learn how to put values onto the channel, and then learn how to take them about. Also, I dislike the phrasing _can be subscribed to_ (passive voice, ending preposition). I can rewrite the entire paragraph if you'd like.